operatorconfig/driverconfig/powerstore/v2.15.0/controller.yaml

```diff
@@ -317,8 +317,6 @@ spec:
               value: "5"
             - name: X_CSI_MULTI_NAS_COOLDOWN_PERIOD
               value: "5m"
-            - name: X_CSI_VOL_PREFIX
```

Comment:
why remove vol prefix support for PowerStore? We allow users to set this value in helm.